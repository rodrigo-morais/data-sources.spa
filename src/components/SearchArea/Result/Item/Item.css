.item {
  margin: 4%;
  border: solid 1px #C7C5C5;
  border-radius: 10px;
  justify-items: stretch;
}

.item .title {
  font-size: 16px;
  font-weight: 500;
  position: relative;
  margin: 25% 0 10% 0;
  width: 100%;
  text-align: center;
  display: block;
}

.item .archived {
  border: solid 1px #ffffff;
  border-radius: 25px;
  background-color: #F34157;
  color: #ffffff;
  font-size: 11px;
  font-weight: 600;
  padding: 2%;
  margin-left: 32%;
}
