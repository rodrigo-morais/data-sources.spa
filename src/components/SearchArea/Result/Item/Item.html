<div class="item">
 <span class="title">{{name}}</span>
 <span class="archived" v-if="archived">ARCHIVED</span>
</div>
