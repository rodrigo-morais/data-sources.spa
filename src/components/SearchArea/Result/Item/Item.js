import Vue from 'vue'
import Component from 'vue-class-component'

export default Component({
  name: 'SearchResult',
  components: { },
  props: { name: String, archived: Boolean },
})(class SearchResult extends Vue {})
