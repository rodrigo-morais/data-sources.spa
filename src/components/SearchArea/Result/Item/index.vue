<template src='./Item.html'></template>

<script src='./Item.js'></script>

<style scoped src='./Item.css'></style>
