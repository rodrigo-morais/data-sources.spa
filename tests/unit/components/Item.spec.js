import { shallowMount } from '@vue/test-utils'
import { createRenderer } from 'vue-server-renderer'
import Item from '@/components/SearchArea/Result/Item/index.vue'

describe('Item.vue', () => {
  it('renders according to design', () => {
    const renderer = createRenderer()
    const wrapper = shallowMount(Item)
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err)
      expect(str).toMatchSnapshot()
    })
  })
})
