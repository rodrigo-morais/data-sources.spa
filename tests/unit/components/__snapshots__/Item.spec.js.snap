// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item.vue renders according to design 1`] = `
<div class="item"><span class="title"></span>
  <!---->
</div>
`;
